packages/story-utils/src/index.ts

```diff
@@ -1,4 +1,2 @@
 export { default as generateSnapshots } from './generateSnapshots';
-export { default as getStories } from './getStories';
-export { default as prepareStory } from './prepareStory';
```

Comment:
As noted, we don't need `prepareStory` anymore because `@storybook/testing-react` now exists.